### PR TITLE
chore: upgrade sdk, add prettier configuration

### PR DESCRIPTION
**Pull request by chrstph-dvx**

### CODE REVIEW

**Review by DZGoldman — APPROVED** [review]
lgtm — cool if we merge this now @Jason-Wanxt ?

**Review by fionnachan — COMMENTED** [review]
when i run `yarn format`, there are still some changes in `main.ts` to be committed
let's also commit those?

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,10 @@ node_modules
 ArbTokenLists
 FullList
 dist
+
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (dewanshparashar) [inline .gitignore:12]: Nit - EOF :/

**Review by dewanshparashar — APPROVED** [review]
LGTM - Very minor nit :)

File: .gitignore (hunk 1 above)
Comment (chrstph-dvx) [reply .gitignore]: changed

**Review by fionnachan — APPROVED** [review]
LGTM

**Pull request merged**